hoodie-common/src/main/avro/HoodieCommitMetadata.avsc

```diff
@@ -37,6 +37,10 @@
                         "name":"numUpdateWrites",
                         "type":["null","long"]
                      },
+                     {
+                        "name":"numInsertsAsUpdateWrites",
```

Comment:
I assumed as long as it has a default value it should be backwards compatible since AVRO does name matching and not index. Hive, however, does index matching so if this schema were written to hive then this would break. On further reading, found this link : https://docs.oracle.com/database/nosql-11.2.2.0/GettingStartedGuide/schemaevolution.html says should be ok

Reply:
if we are sure about that, then fine. I dont remember the rules top off my head . so far we only appended

Reply:
I can append just to be on the safe side...